course/style.css

```diff
@@ -0,0 +1,412 @@
+/*
+Theme Name: Course
+Author: Automattic
+Author URI: https://automattic.com/
+Description: Course is a flexible and modern theme for anyone wanting to share their knowledge. The theme is built with integration with Sensei LMS and is ideal for Sensei users that are creating or selling courses. Style variations with multiple font and color combinations help you craft the perfect look and feel to show off courses and content. The theme can be used without Sensei too.
+Version: 1.2.1
+Requires at least: 6.1
+Tested up to: 6.1
+Requires PHP: 7.2
+License: GNU General Public License v2 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+Text Domain: course
+Tags: block-patterns, eLearning, education, full-site-editing, lms, online courses, sensei, teach, translation-ready
+*/
+
+/*
+ * Font smoothing
+ */
+ body {
+	-moz-osx-font-smoothing: grayscale;
+	-webkit-font-smoothing: antialiased;
+}
+
+/**
+ * Currently table styles are only available with 'wp-block-styles' theme support (block css) thus the following needs to be included
+ * since 'wp-block-styles' aren't used for this theme.
+ */
+.wp-block-table thead {
+	border-bottom: 3px solid;
+}
+
+.wp-block-table tfoot {
+	border-top: 3px solid;
+}
+
+.wp-block-table td,
+.wp-block-table th {
+	padding: var(--wp--preset--spacing--30);
+	border: 1px solid;
+	word-break: normal;
+}
+
+.wp-block-table figcaption {
+	font-size: var(--wp--preset--font-size--small);
+	text-align: center;
+}
+
+ /* Background and text color, padding around for navbar, only for mobile view. Desktop view has separate color */
```

Comment:
Yes, perhaps. I didn't work on the navigation bit so it's unclear to me if some or all of this could be moved into `theme.json`. My preference would be to do any refactoring outside of this particular PR.